azure/services/scalesets/scalesets_test.go

```diff
@@ -265,7 +265,7 @@ func TestReconcileVMSS(t *testing.T) {
 				s.ScaleSetSpec().Return(defaultSpec).AnyTimes()
 				createdVMSS := newDefaultVMSS("VM_SIZE")
 				instances := newDefaultInstances()
-				_ = setupDefaultVMSSInProgressOperationDoneExpectations(s, m, createdVMSS, instances)
+				setupDefaultVMSSInProgressOperationDoneExpectations(s, m, createdVMSS, instances)
```

Comment:
Is this something that `unparam` enforces? I prefer assignment to `_` in general, because it documents that you meant to ignore the error (or other) return value. WDYT?

Reply:
The return value from that function is nowhere used that is why it is enforced. If we want to use the return value somewhere in the future we can modify accordingly. And wherever we do not want we can use the `_` assignment.

Reply:
Copying @sbueringer here in case he has some thoughts.

Reply:
Just my two cents. In general I would also prefer to explicitly assign to _ when a return value is ignored (especially an error).

*But* if I see correctly this PR also removes the return parameter entirely because it's not used anywhere.

I think we have the following options:
* (as is) remove the return parameter because it's unused
* keep the return parameter because we assume it's useful in the future, then we can add ~ `//nolint:unparam // We want to keep the return parameter even though it is currently unused`

Reply:
+1 to removing the `_` assignment since that function is no longer returning a value, we're not actually ignoring a return value